roles/kubevirt-apb/templates/DaemonSet_libvirt.yml

```diff
@@ -0,0 +1,74 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    kubevirt.io: libvirt
+  name: libvirt
+  namespace: kube-system
+spec:
+  template:
+    metadata:
+      labels:
+        kubevirt.io: libvirt
+      name: libvirt
+    spec:
+      containers:
+      - command:
+        - /libvirtd.sh
+        image: kubevirt/libvirt-kubevirt:v0.2.0
```

Comment:
We may not want to hardcode the version.  Let's make all the files with versions .j2 files and render them.